pylint_odoo/test_repo/no_odoo_module/eval_used.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+
+
+def func2(param):
```

Comment:
Ummm maybe something similar to the docblocks? `eval_from_param` and `eval_from_other`?